src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsGeofencingZoneMapper.java

```diff
@@ -51,7 +51,8 @@ private GeofencingZone toInternalModel(GBFSFeature f) {
       LOG.error("Could not convert geofencing zone", e);
       return null;
     }
-    var name = Objects.requireNonNullElseGet(f.getProperties().getName(), () -> fallbackId(g));
+    var nameFromData = f.getProperties().getName().isEmpty() ? null : f.getProperties().getName();
```

Comment:
If the name is null, this would probably lead to npe. I don't know if it can be null.

Reply:
The next line takes care of the null. The issue was that the code was only prepared for the name to be null and did not take into account that it could be an empty string instead.